webserver/include/webserver/pluginhelper.h

```diff
@@ -27,25 +28,27 @@ inline void registerPluginSimple(
   ServiceFactoryT serviceFactory_,
   const std::string& serviceName_)
 {
+  namespace fs = boost::filesystem;
   namespace po = boost::program_options;
 
-  const util::WorkspaceOptions workspaces
-    = util::parseConfigFile(vm_["workspaceCfgFile"].as<std::string>());
-
-  // Create a handler instance for all workspaces.
-  for (const auto& workspace : workspaces)
+  for (fs::directory_iterator it(vm_["workspace"].as<std::string>());
+    it != fs::directory_iterator();
+    ++it)
   {
-    const std::string wsId = workspace.first;
-    const util::WorkspaceOption& wsOpt = workspace.second;
+    std::string project = it->path().filename().native();
+
+    std::string connStr = util::updateConnectionString(
```

Comment:
I parsed a project where the database name(test) and the project name(example) are different:
```cpp
./CodeCompass_parser -d "pgsql:database=test;port=6250" -w /home/<username>/cc/wd -n example
```
After this I started the webserver:
```cpp
./CodeCompass_webserver -w /home/<username>/cc/wd -p 6251 -d "pgsql:database=test;port=6250"
```
The program crashed because database `example` doesn't exist.
To resolve this problem I think we should call this function in the `parser.cpp` and update the `database name` in the connection string too.